src/components/@common/ToolTip/index.tsx

```diff
@@ -1,8 +1,14 @@
-import PropTypes from 'prop-types';
-
+import * as React from 'react';
 import Container from './styles';
 
-function ToolTip({ text, align, isDisabled, children }) {
+interface PropTypes {
+  text?: string;
+  isDisabled?: boolean;
+  align: 'top' | 'bottom' | 'left' | 'right';
+  children: React.ReactNode;
+}
+
+function ToolTip({ text, align, isDisabled, children }: PropTypes) {
```

Comment:
`PropsWithChildren` 라는 타입도 있어용 나중에 고려해보시죠! https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0beca137d8552f645064b8a622a6e153864c66ee/types/react/index.d.ts#L830

```suggestion
function ToolTip({ text, align, isDisabled, children }: PropsWithChildren<PropTypes>) {
```